carma/launch/message.launch.py

```diff
@@ -61,8 +61,12 @@ def open_tunnels():
 
 def generate_launch_description():
 
-    # Open http tunnels with carma cloud
-    open_tunnels()
+    enable_opening_tunnels = rospy.get_param("/enable_opening_tunnels", False)
```

Comment:
This won't work. For one it's a ros1 parameter call not ros2 and second, I don't think there's a guarantee of the parameter being set during file processing. I think the most reliable option would be to pass the argument down the launch chain. 
carma_docker -> carma_src -> message.launch.py